pkgs/development/python-modules/pythonparser/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, python
+, regex
+}:
+
+buildPythonPackage rec {
+  pname = "pythonparser";
+  version = "1.3";
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0qby6w2pcfwxi4w62rly08d9wj32amf6m7rmbrqw0xz6mirq7rrp";
+  };
+
+  patches = [
+    ./no_test_parser_teardown.patch
```

Comment:
You could probably add `coverage` to `propagatedBuildInputs` instead.

Reply:
No, this is an internal grammar coverage module unrelated to the `coverage` package, and the failure is that it assumes a certain directory will exist.

I could create the directory instead of skipping the grammar coverage report, but the result won't end up in the package either way, the only difference would be `GRAMMAR COVERAGE: 99.93%` appearing in the log.